### ISSUE TITLE
[Bug] It loads indefinitely with nearly any action.

### ISSUE THREAD

**Issue by WesleyGi** [IKAJIAN/rtila-releases#7]
Doesn't matter what I do, all I see is the loading icon in the middle of the screen and the app doesn't let me click anything else.

I have to forcibly close it and re-open it constantly to be able to do anything, after such a force-close it did indeed execute the action I asked for it to do which makes this even more annoying.

I can start the Inspector once per run. If It crashes, or I close it, and I want to open it again I get an infinite loading icon until I close/kill the app and re-open it.

**Comment by WesleyGi** [comment]
Much, much worse. After using the tool for a short period of time (Around 30 minutes) it starts to destabilise so heavily beyond all forms of repair.
At least, that's what it looks like at first but it gets even worse beyond that.
Not only does it basically start to self-destruct (without saving data, even erasing saved data) it also starts to completely ruin and destabilise my computer now at this point.
All applications start lock/freeze/crashing one by one, Task Manager refuses to open or end processes, the desktop disappears.
It renders the computer completely unusable, forcing you to restart it however the windows-button doesn't work so you can literally only force the physical off button.

**Comment by IKAJIAN** [comment]
Thanks for your feedback, can you please share which version are you using and what's your hardware configuration?

**Comment by WesleyGi** [comment]
Latest 5.1.2 but it has never truly been stable.

System Model: HP ZBook Power G7 Mobile Workstation
Processor: Intel(R) Core(TM) i7-10750H CPU @ 2.60GHz 2.59 GHz
Ram: 32,0 GB (31,8 GB usable)

**Comment by IKAJIAN** [comment]
That's a very weird behavior, are you using any antivirus software?

**Comment by WesleyGi** [comment]
I am, but every PC here is yet others seem to have no issues with it.

**Comment by IKAJIAN** [comment]
That's correct, after getting your feedback about this behavior we stopped working on other platforms and we focused on debugging what you reported. We did find an issue with a third library used to kill all processes in the process tree, including the root process which caused infinite loops.
Please try this pre-release and let me know if it works for you: https://github.com/IKAJIAN/rtila-releases/releases/tag/v5.2.0

**Comment by IKAJIAN** [comment]
Sorry we updated the binary file, here is the direct download link for Windows: https://github.com/IKAJIAN/rtila-releases/releases/download/v5.2.0/RTILA-Windows-5.2.0.exe

**Comment by IKAJIAN** [comment]
Thanks a lot for your feedback, can you please make a short demo video?

**Comment by WesleyGi** [comment]
I have just force-restarted my computer again after everything froze up when I tried using RTila a second time. It seems the problem has not been solved.
Not sure if I can get recording software, I can currently not record because most other software seems to crash quite quickly in the background.

**Comment by WesleyGi** [comment]
Restarted once more after a clean reïnstall. Issue definitely still there.

**Comment by IKAJIAN** [comment]
What antivirus are you using? if you can't add an exclusion we can try to reproduce same environment for better debugging.

**Comment by WesleyGi** [comment]
We have 4 users here running BitDefender, I am the only one with this issue.

**Comment by IKAJIAN** [comment]
While trying to resolve the issue you reported we did improve how components were called, so thanks a lot for your feedback.

Do all of your users have the same hardware configuration you shared before?

Please note that with the new upgrade all old projects will be lost (still can be recovered by downgrading), so it’s better to export projects as .RTILA files and import them manually when planning to install the new upgrade.

**Comment by WesleyGi** [comment]
We did already note that from the patch notes. Their configurations are different than mine, they're on different machines.

All of them were running 5.1.2 they said, and we've upgraded one system to 5.2.0 now but it's not loading beyond the starting tool, infinitely loading on spawning the GUI.

**Comment by WesleyGi** [comment]
His system is working now, it just took a long while to start properly. His version seems to be working correctly, no crashes, freezes, hangups or anything else.

Small feature request, being able to pin inspection/automation to stay open.
